verl/workers/fsdp_workers.py

```diff
@@ -125,6 +125,7 @@ def __init__(self, config: DictConfig, role: str):
                 self.config.actor.ppo_micro_batch_size //= (self.device_mesh.shape[0] //
                                                             self.ulysses_sequence_parallel_size)
                 self.config.actor.ppo_micro_batch_size_per_gpu = self.config.actor.ppo_micro_batch_size
+                assert self.config.actor.ppo_mini_batch_size % self.config.actor.ppo_micro_batch_size_per_gpu == 0
```

Comment:
We also have to encore self.config.actor.ppo_mini_batch_size >= n_gpus * self.config.actor.ppo_micro_batch_size_per_gpu

Reply:
Is it necessary? The mini_batch_size here is already normalized, then if self.config.actor.ppo_mini_batch_size < n_gpus * self.config.actor.ppo_micro_batch_size_per_gpu, the above line will not get 0